src/app/footer-navbar/footer-navbar.component.html

```diff
@@ -1,12 +1,20 @@
-<footer>
-  <div class="left-side">
-      <a routerLink="/about" routerLinkActive="active">About</a>
-      <a href="//blog.fossasia.org">Blogs</a>
-      <a href="//github.com/fossasia/susper.com">Code</a>
+<nav class="navbar navbar-default footer navbar-fixed-bottom" role="navigation">
+  <div class="navbar-header">
+    <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+      <span class="icon-bar"></span>
+      <span class="icon-bar"></span>
```

Comment:
@nikhilrayaprolu The thing is we do not want those three lines to appear, since we donot want a dropdown menu on small screens. It is impossible to get the same effect as loklak.net with the bootstrap classes so we need to remove all of them. (See #180) We need to have the classes footer and left and right. We need to make changes in the css of these classes to push the bar to the bottom. I will make a PR for this soon.